tests/test_pyladies_cz.py

```diff
@@ -0,0 +1,42 @@
+import os
+import sys
+import importlib.util
+
+import pytest
+
+def register_app_as_module():
```

Comment:
Ak niekto vie lepsi sposob ako to loadnut, bolo by asi dobre ak by ste sa ozvali... :D